src/Cooked/MockChain/GenerateTx.hs

```diff
@@ -61,7 +63,7 @@ data Context where
     Context
 
 instance Default Context where
-  def = Context 0 mempty def Map.empty Map.empty Map.empty
+  def = Context 0 mempty (wallet 1) def Map.empty Map.empty Map.empty
```

Comment:
That's a good point. Since we need this default instance for the Context because we invoke the generation functions sometimes with only limited info, I can't remove it. What I will do however is make it optional and have `Nothing` as the default value.

Reply:
I was suggesting a Maybe too in a previous version of this comment (that I cut short because it was too long). Glad to see we agree.